Makefile

```diff
@@ -2,14 +2,61 @@
 
 SHELL := /bin/bash
 MAKEFILE_DIR = $(patsubst %/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
+
+# from PYTHON_VERSION depends which `./ci/requirements-$PYTHON_VERSION-dev` file
+# will be used for creating ibis image (see for additional info: `./ci/Dockerfile.dev`
```

Comment:
so these comments are fine, but maybe my question wasn't clear. we use the PYTHON variable inside Dockerfile.dev and NOT PYTHON_VERSION. I think we should simply fix this or is there a reason that is not obvious?